src/swedish-pancakes.md

```diff
@@ -0,0 +1,49 @@
+# Swedish Pancakes
+
+![Swedish Pancakes](pix/swedish-pancakes.webp)
+
+## Ingredients
+
+Small batch(6-8):
+
+- Eggs, beaten: 3
```

Comment:
This is kinda inconsistent with most recipes. Usually the number comes before the ingredients but it's probably fine.